q2_types/per_sample_sequences/__init__.py

```diff
@@ -23,10 +23,15 @@
                       PairedEndFastqManifestPhred33V2,
                       PairedEndFastqManifestPhred64V2,
                       QIIME1DemuxFormat, QIIME1DemuxDirFmt,
-                      SampleIdIndexedSingleEndPerSampleDirFmt)
+                      SampleIdIndexedSingleEndPerSampleDirFmt,
+                      MultiMAGSequencesDirFmt, MultiMAGManifestFormat,
+                      ContigSequencesDirFmt, MultiBowtie2IndexDirFmt,
+                      BAMFormat, BAMDirFmt, MultiBAMDirFmt,
+                      MultiFASTADirectoryFormat)
```

Comment:
Looks like `MultiFASTADirectoryFormat` wasn't in the __init__ under per_sample_data. Still examining but adding this note here to reference later.